spec/services/handover_date_service_spec.rb

```diff
@@ -3,32 +3,36 @@
 describe HandoverDateService do
   describe '#handover_start_date' do
     context 'when NPS' do
-      context 'with early allocation' do
-        let(:offender) { OpenStruct.new(nps_case?: true, early_allocation?: true) }
+      let(:offender) {
+        OpenStruct.new indeterminate_sentence?: indeterminate,
+                       nps_case?: true,
+                       automatic_release_date: release_date,
+                       tariff_date: tariff_date
+      }
 
-        it 'is not set' do
-          expect(described_class.handover(offender).start_date).to be_nil
+      let(:release_date) { Date.new(2020, 8, 30) }
```

Comment:
Just trying to understand what some of these dates mean. Does it matter that this is in the future?